### PR TITLE
Add the information aout the bootstrap with pricing details AB-8617

### PR DESCRIPTION
**Pull request by andreorvalho**
[JIRA TICKET](https://autobutler.atlassian.net/browse/AB-8617)

### CODE REVIEW

File: source/includes/_data_bootstrap.md [hunk 1]
```diff
@@ -97,6 +116,7 @@ user.subMechanics.id             | integer       | no            | The id of a s
 user.subMechanics.name           | string        | no            | The name of a specific submechanic
 user.subMechanics.zip            | string        | no            | ZIP code of the area where specific submechanic is located
 user.subMechanics.city           | string        | no            | Name of the city where specific submechanic is located
+pricingDetails                   | object        | no            | A list of pricing details from the config file
```
Comment (larssg) [inline source/includes/_data_bootstrap.md]: I'd like some more info here. Also, you need to remove user.vatFactor above.

Comment (andreorvalho) [inline source/includes/_data_bootstrap.md]: hum what kind of information, they are 4 different values I am not sure how to describe it differently: 

vatFactor, serviceFee,  serviceFeeTreshold ,minimumOfferPrice

I will remove the vat factor from the user.

Comment (larssg) [inline source/includes/_data_bootstrap.md]: Exactly - would be nice to have them listed in the documentation

**Comment by andreorvalho** [comment]
@larssg fixed

File: source/includes/_data_bootstrap.md [hunk 2]
```diff
@@ -97,6 +116,7 @@ user.subMechanics.id             | integer       | no            | The id of a s
 user.subMechanics.name           | string        | no            | The name of a specific submechanic
 user.subMechanics.zip            | string        | no            | ZIP code of the area where specific submechanic is located
 user.subMechanics.city           | string        | no            | Name of the city where specific submechanic is located
+pricingDetails                   | object        | no            | A list of pricing details from the config file. Includes: vat_factor, service_fee, service_fee_treshold ,minimum_offer_price
```
Comment (larssg) [inline source/includes/_data_bootstrap.md]: I was hoping you'd do it like above :-)

Comment (andreorvalho) [inline source/includes/_data_bootstrap.md]: In camel case? or on the description?

Comment (andreorvalho) [inline source/includes/_data_bootstrap.md]: @larssg could you give an example of what you are expecting?

Comment (larssg) [inline source/includes/_data_bootstrap.md]: A line per sub field, as above.

Comment (andreorvalho) [inline source/includes/_data_bootstrap.md]: it doesn't seem like there is a good solution for this: 
http://stackoverflow.com/questions/19950648/how-to-write-lists-inside-a-markdown-table

Comment (dipth) [inline source/includes/_data_bootstrap.md]:
```
pricingDetails.vat_factor           | bla        | bla            | bla
pricingDetails.service_fee          | bla        | bla            | bla
pricingDetails.service_fee_treshold | bla        | bla            | bla
pricingDetails.minimum_offer_price  | bla        | bla            | bla
```

Comment (dipth) [inline source/includes/_data_bootstrap.md]: The practice we're using is having one table row per json attribute.
If an attribute contains a sub-object that itself contains attributes, then you add individual table rows with the full path to the attribute with `.` as seperators.

```
pricingDetails                      | object     | bla            | An object containing the pricing details
pricingDetails.vat_factor           | bla        | bla            | bla
pricingDetails.service_fee          | bla        | bla            | bla
pricingDetails.service_fee_treshold | bla        | bla            | bla
pricingDetails.minimum_offer_price  | bla        | bla            | bla
```

**Comment by andreorvalho** [comment]
@dipth fixed

**Pull request merged**